examples/hello_world/hello_world_library.gdnlib

```diff
@@ -2,6 +2,7 @@
 
 X11.64="../../target/debug/libhello_world.so"
 OSX.64="../../target/debug/libhello_world.dylib"
+Windows.64="res://../../target/debug/hello_world.dll"
```

Comment:
Does this work without the `res://` or does it also work if we put the `res://` everywhere? It would be nice if this was the same everywhere

Reply:
I cannot easily test on other platforms... Could you do a quick and dirty test on yours?

But yes, I confirmed that it DOES NOT work without res:// -- at least for Windows and hello_world. I think I confirmed it for scene_create too at the time.

It makes some sense, windows uses \annoying\non\unix\slashes not /var/awesome/linux/ones.

Reply:
Sure, I will give it a quick check!

Reply:
Yes it works, if you could add the `res://` prefix to the Linux and OSX entries as well, that would be amazing. If not I can do it in a follow-up commit.